ogc-example/src/main/scala/geotrellis/server/ogc/conf/OgcServiceConf.scala

```diff
@@ -28,12 +27,12 @@ import geotrellis.server.ogc.wmts.GeotrellisTileMatrixSet
  */
 sealed trait OgcServiceConf {
   def layerDefinitions: List[OgcSourceConf]
-  def layerSources(simpleSources: List[SimpleSource]): List[OgcSource] = {
+  def layerSources(simpleSources: List[SimpleSource]): OgcSourceRepository = {
     val simpleLayers =
-      layerDefinitions.collect { case ssc@SimpleSourceConf(_, _, _, _) => ssc.model }
+      layerDefinitions.collect { case ssc @ SimpleSourceConf(_, _, _, _) => ssc.models }.flatten
```

Comment:
This looks like it may result in each layer being listed once per time-stamp in GetCapabilities ?